NumPy-survey-questionnaire.md

```diff
@@ -87,6 +95,9 @@
 <h4>What other significant changes would you like to see in NumPy?</h4>
 <p><i>allow text entry</i></p>
 <br>
+<h4>What was the most difficult or confusing (recent) experience with NumPy?</h4>
+<p><i>allow text entry</i></p>
+
 <h4>Do you think your organization may be interested in funding NumPy?</h4>
```

Comment:
Maybe another good question, is which parts of numpy they normally use. `Polynomials`, `FFT`, `linalg`, `random`, (financial?). Or even more abstract things like `advanced indexing/indexing with arrays`

Reply:
Survey Design Query:  Are some/all of these Questions optional/mandatory for the survey taker?

I see three lines of questions:
1. About the user (Org, skill level, location, gender, demographic etc etc.)
2. About NumPy usage (Version, Use case etc.)
3. NumPy Roadmap & Contribution (Which feature(s) are highly in use, what is of high priority in terms of various NumPy functionalities, docs, website info etc and whether devs/orgs would be interested in contributing their time or money to NumPy)

Regarding #2 above, it would be nice to have the following additional questions:

* How long have you used NumPy?
 - <6mths
 - 6mths - 2 yrs
 -  2 yrs or more

* How often do you use NumPy?
 - Daily
 - Once a week
 - Once a month
 - Once in 6 months

* Would you recommend NumPy to other users? [Yes/No] If No - provide a text field (optional) for the reason of not recommending.

Reply:
Have to incorporate it, but before I forget. We should ask if people feel numpy is too slow, or too fast (breaking things) moving as a project. Also what people feel is a good deprecation time frame.